products/dce.md

```diff
@@ -0,0 +1,48 @@
+---
+title: DaoCloud Enterprise
+category: server-app
+tags: daocloud
+permalink: /dce
+releasePolicyLink: https://docs.daocloud.io/en/dce/support.html
+changelogTemplate: https://docs.daocloud.io/en/dce/dce-rn/20240530.html
+releaseDateColumn: true
+eolColumn: Core Support
+eoesColumn: true
+
+auto:
+  methods:
+  -   custom: https://github.com/DaoCloud/DaoCloud-docs.git
```

Comment:
The tags at https://github.com/DaoCloud/DaoCloud-docs/tags don't seem to match the OS versions.

```suggestion
```